docs/shared/monorepo-ci-github-actions.md

```diff
@@ -136,22 +133,18 @@ jobs:
       - name: Run commands in parallel
```

Comment:
I know it wasn't part of the original PR - but can we change the `run: npx nx-cloud start-ci-run` to `run: npx nx-cloud start-ci-run --stop-agents-after="build"` please? 🙏 

you'll see it as an example on: https://nx.dev/recipes/ci/monorepo-ci-circle-ci#configuring-ci-using-circleci-and-nx